core/src/main/java/com/bumble/appyx/core/node/Node.kt

```diff
@@ -80,6 +84,13 @@ abstract class Node(
         });
     }
 
+    protected suspend inline fun <reified T> executeWorkflow(
+        crossinline action: () -> Unit
+    ): T = withContext(lifecycleScope.coroutineContext + Dispatchers.Main) {
```

Comment:
`Lifecycle.lifecycleScope` is already `Main`.

Reply:
it uses `Main.immediate` which can change executing order. `attachWorkflow` doesn't work without `Dispatchers.Main` because right after after calling action we search for a certain element. Because the order is changed children won't be synced at this time. 

Execute workflow works without explicitly setting `Dispatchers.Main` but for the sake of consistency I'd keep it

Reply:
Removed. Now we perform an action and waiting for the children to be in sync with routing source. After 1 second throw an exception if desired Node was not found